src/app/api/auth/[kindeAuth]/route.ts

```diff
@@ -0,0 +1,13 @@
+import { handleAuth } from '@kinde-oss/kinde-auth-nextjs/server'
+
+type Params = {
+  params: {
+    kindeAuth: string,
+  },
+}
+
+export const GET = async (request: object, { params }: Params) => {
```

Comment:
I was gna say can we name this better but I see its just straight from the docs so meh